mozilla/kuma
# [DO NOT MERGE] Bug 1170235 compat tables

How to test:
- download this branch and compile your style sheets and compress assets
- create a macro from this: https://gist.github.com/darkwing/6fccdfd9852f338799eb
- create a page with the macro on it

Still to do:
- [ ] Icon font not appearing (may be unrelated issue)
- [x] Space in the table slides open but the history drawer itself just appears.
- [ ] Space in the table slides closed but the history drawer itself just dissipears.
- [x] The heading is also 'sliding open' when the history draw is opened in tablet/flex view. (I'll look at a fix)
- [x] Need explicit way to trigger thick border between support cells when medium switches.
- [x] Add `.bc-medium-border` to the macro

Where should we identify problems with the macro? The contents, legend, and title of many of the icons/abbreviations need to be changed.
